src/runtime/mod.rs

```diff
@@ -337,7 +337,7 @@ mod tests {
             ..Default::default()
         };
         let test_dir = tempfile::tempdir().expect("failed to create tmp test dir");
-        let spec_path = test_dir.into_path().join("config.json");
+        let spec_path = test_dir.keep().join("config.json");
```

Comment:
This fix was made because the CI failed.
According to the documentation, into_path is deprecated, and it recommends using keep instead.
Reference: https://docs.rs/tempfile/latest/tempfile/struct.TempDir.html#method.into_path